src/components/KMultiselect/KMultiselectItems.vue

```diff
@@ -63,28 +101,31 @@ const groups = computed((): string[] => [...new Set((props.items?.filter(item =>
 
 const getGroupItems = (group: string) => props.items?.filter(item => item.group === group)
 
-const onKeyDown = (event: Event) => {
-  const { target, key } = event as KeyboardEvent
+const itemsContainer = ref<HTMLDivElement | null>(null)
+
+const setItemFocus = (): void => {
+  const firstSelectableItem = itemsContainer.value?.querySelectorAll<HTMLButtonElement>('.multiselect-item button:not([disabled])')[0]
```

Comment:
Since we are querying the first result here we can use this:

```suggestion
  const firstSelectableItem = itemsContainer.value?.querySelector<HTMLButtonElement>('.multiselect-item button:not(:disabled)')
```

nit: I would prefer pseudo classes over attribute selectors as they are more semantically precise (`[disabled]` &rarr; `:disabled`)